colab/accounts/templates/accounts/manage_subscriptions.html

```diff
@@ -26,11 +26,29 @@ <h3 class="panel-title">{{ email }}</h3>
               </div>
             {% endfor %}
           </div>
+          <div class="panel-footer text-center">
```

Comment:
@matheus-fga , this part was moved to https://github.com/colab/colab/blob/detach_super_archives/colab/accounts/templates/accounts/subscription_lists.html, after @italopaiva added search by listname